### DISCUSSION TITLE
Interlaced graphics

### DISCUSSION THREAD

**Discussion by User32sys** [hneemann/Digital#1328] [Q&A] [answered]
I have updated my java version and did a recompile of the source code. Now my user interface shows interlaced graphics which is very annoying. Is there a way to get rid of this?

**Comment by hneemann** [comment]
I've never heard that before! Can you attach a screenshot?

**Comment by User32sys** [comment, marked as answer]
Deleted all files and recompiled again but now it seems ok. I noticed large icons in the component view and large bold fonts when the problem occured. Maybe the font settings through me off.
